modules/src/ics07_tendermint/client_state.rs

```diff
@@ -131,6 +140,20 @@ impl ClientState {
     pub fn expired(&self, elapsed: Duration) -> bool {
         elapsed > self.trusting_period
     }
+
+    /// Helper method to produce a
+    /// [`tendermint_light_client::light_client::Options`] struct for use in
+    /// Tendermint-specific light client verification.
+    pub fn light_client_options(&self) -> Result<Options, Error> {
```

Comment:
Maybe call this `to_light_client_options`? I'm always confused by these prefix naming conventions, so will put the lints here: https://rust-lang.github.io/rust-clippy/master/index.html#wrong_self_convention to help inform the discussion.

Reply:
I think `as_light_client_options` makes more sense, since we're not consuming `self`.